app/assets/stylesheets/components/_scoreboard_player_card.scss

```diff
@@ -81,6 +100,10 @@
   color: $royal_blue;
 }
 
+.mb-1 img {
```

Comment:
Is there a better way to do this than making it based on a common Bootstrap class?

Reply:
I can call it on text-center class instead, is that a bootstrap class?